common/djangoapps/util/enterprise_helpers.py

```diff
@@ -19,7 +23,7 @@ def enterprise_enabled():
     """
     Determines whether the Enterprise app is installed
     """
-    return 'enterprise' in settings.INSTALLED_APPS
+    return 'enterprise' in settings.INSTALLED_APPS and not FAILED_ENTERPRISE_IMPORT
```

Comment:
Since `enterprise` is an optional app defined in `OPTIONAL_APPS`, under what circumstances would it be in `settings.INSTALLED_APPS` but not be able to be imported?